docs/reference/config_base.md

```diff
@@ -0,0 +1,6 @@
+# Base configuration file
+
+MaxText uses a base configuration file to set default values for various parameters. This file is located at [`src/MaxText/configs/base.yml`](https://github.com/AI-Hypercomputer/maxtext/blob/main/src/MaxText/configs/base.yml) in the MaxText repository, and is shown below.
+
+```{literalinclude} ../../src/MaxText/configs/base.yml
```

Comment:
This creates a nice page with the full contents of base.yml. If you don't think this is useful, I can remove.